src/views/home/components/bridge-form/bridge-form.view.tsx

```diff
@@ -42,7 +43,9 @@ interface SelectedChains {
   to: Chain;
 }
 
-const BridgeForm: FC<BridgeFormProps> = ({ account, formData, resetForm, onSubmit }) => {
+const BridgeForm: FC<BridgeFormProps> = ({ account, onSubmit }) => {
+  const { state: routerState } = useLocation();
```

Comment:
I don't like to use the `useLocation` hook inside a component that it's not a route. This is not the case by the moment, but imagine that you use this component under another route in the app. They parser could fail and an error would appear.

I'll move this to the `Home` component.